inst/qml/uniDimBayesian.qml

```diff
@@ -27,178 +27,204 @@ Form
   	{
```

Comment:
Please use tabs for indentation (also where there were already spaces). Also, if you change the names in the QML files then I think you also need to list the changes in a separate file to ensure backward compatibility. Perhaps @JorisGoosen can tell us more about that.

Reply:
ok so in the short term it's probably best to stick to the old names in the QML file

Reply:
I think I will have added support for it some time in the coming week(s), so you could also wait for that to be finished.

Reply:
I can wait. What you are talking about will also be concerning Description.qml?

Reply:
No that was simply the first part of that issue but was implemented. That is why you use "Description.qml" as opposed to "description.json".
But alright, I am actually working on the upgrades now and I think it would be practical for you to "watch" https://github.com/jasp-stats/INTERNAL-jasp/issues/686 so that you get notified when I am done with it.

Reply:
so now that @JorisGoosen closed his issue, how can I ensure that there will be backward compatibility for new names in my .qml files?

Reply:
I think that https://github.com/jasp-stats/jasp-desktop/pull/4460 needs to be merged and then @JorisGoosen can probably tell you how to do it.

Reply:
Yeah the PR needs to be merged, but you could already work on the renaming and I've written a lovely manual (If I say so myself) which you can already read at https://github.com/JorisGoosen/jasp-desktop/blob/UpgradesQML/Docs/development/jasp-upgrade-qml.md

Reply:
(And if anything there is unclear, let me know in a PM on slack what is and we can see about improving it)

Reply:
Ok, the upgrades are in and some nightlies are building, so you can start working on the Upgrades.qml for your changes